src/components/SearchBox/SearchBox.tsx

```diff
@@ -79,7 +79,7 @@ class SearchBox extends Component {
     onChange(event);
   };
 
-  componentWillReceiveProps(nextProps) {
+  public componentWillReceiveProps(nextProps: SearchBoxProps) {
```

Comment:
Same here, I'm wondering if we can make it private. But now I'm starting to doubt :D

Reply:
React lifecycle methods must be set as public, since React will internally call them :slightly_smiling_face: